src/app/profile/api/getProfileList.ts

```
@@ -5,7 +5,9 @@ export async function getProfileList({
   year = [],
   university = [],
   grade = [],
+  cursorId,
   limit: limit,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)